packages/wrangler/src/api/pages/publish.tsx

```diff
@@ -3,6 +3,8 @@ import { tmpdir } from "node:os";
 import { join, resolve as resolvePath } from "node:path";
 import { cwd } from "node:process";
 import { File, FormData } from "undici";
+import { parseHeaders } from "../../../../pages-shared/metadata-generator/parseHeaders";
+import { parseRedirects } from "../../../../pages-shared/metadata-generator/parseRedirects";
```

Comment:
Should I make these use `@cloudflare/pages-shared` instead? Not sure if that could cause it to ever have an older version..